src/components/Common/UI/Table/Table.module.scss

```diff
@@ -21,8 +22,19 @@
     :global(.react-aria-TableHeader) {
       th {
         color: var(--g-colorBodyContent);
-        font-weight: 400;
-        font-size: var(--g-fontSizeSmall);
+        font-weight: 500;
```

Comment:
Totally cool if you don't know this, but I wonder if we have a weight variable that can cover this?